### PR TITLE
Cleaning up telemetry

### PR DESCRIPTION
**Pull request by yafimvo**
1. `CONTRIBUTING.md` updated.
2. Documentation added to `SKLearnEvaluationLogger`
3. Old  `telemetry.log_call` call replaced with `SKLearnEvaluationLogger.log`.

Issue #68

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,33 +67,56 @@ Each function must have a corresponding test. If the function has parameters tha
...
+                        "target_names": "None",
+                        "normalize": "False",
+                        "cmap": "None",
+                        "ax": "None"
+                    }
+          }
+        }
+```
 
-    Parameters
```
Comment (edublancas) [inline CONTRIBUTING.md:81]: looks like this whole section was removed. did you rebase?

**Review by edublancas — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (yafimvo) [reply CONTRIBUTING.md]: Fixed it. My forked repo wasn't synced with the latest version.

File: src/sklearn_evaluation/plot/classification.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def __init__(self, a, b, target_names) -> None:
 
 class ConfusionMatrix(Plot):
 
-    @telemetry.log_call('sklearn-evaluation-ConfusionMatrix')
+    @SKLearnEvaluationLogger.log(feature='plot', action="confusion_matrix__init__")
```
Comment (idomic) [inline src/sklearn_evaluation/plot/classification.py:56]: let's use: `confusion-matrix-init` for consistency

**Review by idomic — COMMENTED** [review]
Added comment

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -100,3 +97,67 @@ def my_plotting_function(y_true, y_pred, ax=None):
     """
     pass
 ```
+
+## Telemetry : Monitoring the state of `sklearn-evaluation`
+
+Use `SKLearnEvaluationLogger` decorator to generate logs
+
+```python
```
[the remark this answers is not in the document]
Comment (edublancas) [reply CONTRIBUTING.md:105]: can you address this?

**Review by edublancas — CHANGES_REQUESTED** [review]

**Comment by edublancas** [comment]
there is one file with conflicts (classification.py) @yafimvo, please address it

**Review by edublancas — APPROVED** [review]

**Pull request merged**